languages/abbreviations/fr.json

```diff
@@ -0,0 +1,570 @@
+        "arrêt": "Arr.",
+        "zone": "zon.",
+        "municipaux": "Munic.",
+        "grand": "Gd",
+        "janvier": "Janv.",
+        "fondateur": "Fond.",
+        "première": "1re",
+        "municipale": "Munic.",
+        "direction": "Dir.",
+        "anonyme": "Anon.",
+        "départementale": "Dépt",
+        "moyens": "Moy.",
+        "novembre": "Nov.",
+        "jardin": "Jard.",
+        "petites": "Pet.",
+        "privé": "Priv.",
+        "centres": "Ctres",
+        "forestier": "Forest.",
+        "xiv": "14",
+        "africaines": "Afric.",
+        "sergent": "Sgt",
+        "européenne": "Eur.",
+        "privée": "Priv.",
+        "café": "Cfé",
+        "xix": "19",
+        "hautes": "Htes",
+        "major": "Mjr",
+        "vendredi": "ve.",
+        "municipalité": "Munic.",
+        "sous-préfecture": "Ss-préf.",
+        "spéciales": "Spéc.",
+        "secondaires": "Second.",
+        "viie": "7e",
+        "moyenne": "Moy.",
+        "commerciale": "Commerc.",
+        "région": "Rég.",
+        "américaines": "Amér.",
+        "américains": "Amér.",
+        "service": "Sce",
+        "professeur": "Prof.",
+        "départemental": "Dépt",
+        "hôtels": "Hôt.",
+        "mondiales": "Mond.",
+        "ire": "1re",
+        "caporal": "Capo.",
+        "militaire": "Milit.",
+        "lycée d'enseignement professionnel": "LEP",
+        "adjudant": "Adj.",
+        "médicale": "Méd.",
+        "conférences": "Confér.",
+        "universelle": "Univ.",
+        "xiie": "12e",
+        "supérieures": "Sup.",
+        "naturel": "Natur.",
+        "société nationale": "SN",
+        "hospitalier": "Hosp.",
+        "culturelle": "Cult.",
+        "américain": "Amér.",
+        "son altesse royale": "S.A.R.",
+        "infirmière": "Infirm.",
+        "viii": "8",
+        "fondatrice": "Fond.",
+        "madame": "Mme",
+        "métropolitain": "Métrop.",
+        "ophtalmologues": "Ophtalmos",
+        "xviie": "18e",
+        "viiie": "8e",
+        "commerçante": "Commerç.",
+        "centre d'enseignement du second degré": "CES",
+        "septembre": "Sept.",
+        "agriculteur": "Agric.",
+        "xiii": "13",
+        "pontifical": "Pontif.",
+        "cafétéria": "Cafét.",
+        "prince": "Pce",
+        "vie": "6e",
+        "archiduchesse": "Archid.",
+        "occidental": "Occ.",
+        "spectacles": "Spect.",
+        "camping": "Camp.",
+        "métro": "Mº",
+        "arrondissement": "Arrond.",
+        "viticole": "Vitic.",
+        "ii": "2",
+        "siècle": "Si.",
+        "chapelles": "Chap.",
+        "centre": "Ctre",
+        "sapeur-pompiers": "Sap.-pomp.",
+        "établissements": "Étabts",
+        "société anonyme": "SA",
+        "directeurs": "Dir.",
+        "vii": "7",
+        "culturel": "Cult.",
+        "central": "Ctral",
+        "métropolitaine": "Métrop.",
+        "administrations": "Admin.",
+        "amiraux": "Amir.",
+        "sur": "\/",
```

Comment:
These backslashes probably shouldn’t be in here, should they? If I’m not mistaken, “sur” would be “[s/](https://en.wiktionary.org/wiki/s/#French)”, not just “/”.